projects/components/src/table/table.component.scss

```diff
@@ -67,7 +67,11 @@
 
   .header-cell-renderer {
     width: 100%;
-    padding: 10px 12px;
+    padding: 10px 12px 10px 7px;
```

Comment:
Where's the odd value (-5px) from? Aren't we moving 4px in the data cell?

Reply:
I could do it, I understand that visually it looks a little better this way.

Reply:
I misunderstood - the issue is that the data was moved over 8px (4 margin, 4 padding) and the header was moved over 17px (5px from the resize handle - 2px padding, 1px width - except in the first col) and 12px padding from this style.

With your change, it's now 12px consistently in the header, and 12px consistently in data - right?

Reply:
Yes. It's consistently now.